spec/acceptance/should_create_task_spec.rb

```diff
@@ -251,4 +251,56 @@
       expect(result.stdout).to match(%r{Last\sTUE})
     end
   end
+
+  it 'creates a task with synchronisation disabled: taskscheduler_api2', tier_high: true do
+    pp = <<-MANIFEST
+    scheduled_task {'#{taskname}':
+      ensure      => present,
+      compatibility => 1,
+      command     => 'c:\\\\windows\\\\system32\\\\notepad.exe',
+      arguments   => "foo bar baz",
+      working_dir => 'c:\\\\windows',
+      trigger => {
+        schedule            => daily,
+        start_time          => '12:00',
+        disable_synchronise => true,
+      },
+      provider    => 'taskscheduler_api2'
+    }
+    MANIFEST
+    apply_manifest(pp, catch_failures: true)
+
+    # Ensure it's idempotent
+    # apply_manifest(pp, catch_changes: true)
```

Comment:
Can this be deleted?

Reply:
Removed and added a note to the type to show that it is non-idempotent